<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="d3df15e7-81d1-42f8-96b5-f8fbc47e7442" name="Oleg@OLEGPC 2016-03-22 17:06:37" runUser="OlegPC\Oleg" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Локальные" id="04b1999a-1c84-4bcd-a2b7-426cafe90866">
    <Description>Это - параметры по умолчанию для локального тестового запуска.</Description>
    <Deployment enabled="false" runDeploymentRoot="Oleg_OLEGPC 2016-03-22 17_06_37" />
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2016-03-22T17:06:37.9055788+03:00" queuing="2016-03-22T17:06:38.1155790+03:00" start="2016-03-22T17:06:38.2555792+03:00" finish="2016-03-22T17:06:39.3955808+03:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="TestPayOrder" storage="d:\tstbank\orderapplication2\testproject1\bin\debug\testproject1.dll" id="96dccff5-4769-49cd-d1f1-93a65ee2ecb3">
      <Execution timeOut="1800000" id="7c8d58c1-a572-4e04-8e71-0c9fdd2b16a6" />
      <TestMethod codeBase="D:/tstbank/OrderApplication2/TestProject1/bin/Debug/TestProject1.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="TestProject1.UnitTest1, TestProject1, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="TestPayOrder" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Отсутствующие в списке результаты" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="Все загруженные результаты" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="96dccff5-4769-49cd-d1f1-93a65ee2ecb3" executionId="7c8d58c1-a572-4e04-8e71-0c9fdd2b16a6" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="7c8d58c1-a572-4e04-8e71-0c9fdd2b16a6" testId="96dccff5-4769-49cd-d1f1-93a65ee2ecb3" testName="TestPayOrder" computerName="OLEGPC" duration="00:00:00.9827423" startTime="2016-03-22T17:06:38.3055793+03:00" endTime="2016-03-22T17:06:39.3355808+03:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="7c8d58c1-a572-4e04-8e71-0c9fdd2b16a6">
      <Output>
        <ErrorInfo>
          <Message>Метод проверки TestProject1.UnitTest1.TestPayOrder выдал исключение: Moq.MockException: 
Expected invocation on the mock exactly 2 times, but was 0 times: m =&gt; m.Pay(It.IsAny&lt;Product&gt;())
No setups configured.
No invocations performed..</Message>
          <StackTrace>    в Moq.Mock.ThrowVerifyException(MethodCall expected, IEnumerable`1 setups, IEnumerable`1 actualCalls, Expression expression, Times times, Int32 callCount)
   в Moq.Mock.VerifyCalls(Interceptor targetInterceptor, MethodCall expected, Expression expression, Times times)
   в Moq.Mock.Verify[T](Mock`1 mock, Expression`1 expression, Times times, String failMessage)
   в Moq.Mock`1.Verify(Expression`1 expression, Times times)
   в TestProject1.UnitTest1.TestPayOrder() в D:\tstbank\OrderApplication2\TestProject1\UnitTest1.cs:строка 43
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>